etc/kayobe/environments/ci-multinode/kolla/globals.yml

```diff
@@ -25,3 +25,12 @@ octavia_provider_agents: "ovn"
 
 # Manila CephFS configuration
 manila_cephfs_filesystem_name: manila-cephfs
+
+# Designate configuration
+designate_ns_record:
+  - "ns1.mos.{{ root_domain | default(example.com) }}"
+  - "ns2.mos.{{ root_domain | default(example.com) }}"
+  - "ns3.mos.{{ root_domain | default(example.com) }}"
```

Comment:
```suggestion
  - "ns1.mos.{{ root_domain | default('example.com') }}"
  - "ns2.mos.{{ root_domain | default('example.com') }}"
  - "ns3.mos.{{ root_domain | default('example.com') }}"
```

Reply:
Do these need to be resolvable?

Reply:
The name servers don't need to be resolvable. This would be important if we wanted/needed authoritative results however the authoritative names for `sms-lab.cloud` are the controllers on sms-lab which cannot reach the multinode controllers.